addons/sourcemod/scripting/MusicName.sp

```diff
@@ -264,17 +264,15 @@ public Action Hook_AmbientSound(char sample[PLATFORM_MAX_PATH], int &entity, flo
 			CPrintToChat(client, "%t %t", "Chat Prefix", "Now Playing", g_sCurrentSong);
 		}
 
-		DataPack hSongData = new DataPack();
-		hSongData.WriteString(sFileName);
-		RequestFrame(ClearPrintedAlready, hSongData);
+		RequestFrame(ClearPrintedAlready, sFileName);
 	}
 	return Plugin_Continue;
 }
 
-public void ClearPrintedAlready(DataPack hSongData)
```

Comment:
If `char sFileName[PLATFORM_MAX_PATH]` is the param, then don't think we need to declare it again in Line 274.